ilastik/applets/labeling/labelingGui.py

```diff
@@ -48,6 +48,8 @@
 
 from ilastik.applets.labeling.labelingImport import import_labeling_layer
 
+from volumina.api import \
```

Comment:
I don't know if you committed some auto changes here in this file in general (by some linter)... But our current agreement is not to do such changes because it's easier to navigate file history.
In any case I would also _always_ opt for parentheses for multi-line imports such as
```python
from volumina.api import (
    GrayscaleLayer, ColortableLayer, LazyflowSinkSource
)
```